lib/manager/github-actions/extract.ts

```diff
@@ -1,30 +1,50 @@
+import * as githubTagsDatasource from '../../datasource/github-tags';
 import { logger } from '../../logger';
 import * as dockerVersioning from '../../versioning/docker';
+import * as looseVersioning from '../../versioning/loose';
 import { PackageDependency, PackageFile } from '../common';
 import { getDep } from '../dockerfile/extract';
 
 export function extractPackageFile(content: string): PackageFile | null {
   logger.debug('github-actions.extractPackageFile()');
   const deps: PackageDependency[] = [];
   for (const line of content.split('\n')) {
-    // old github actions syntax will be deprecated on September 30, 2019
-    // after that, the first line can be removed
-    const match =
-      /^\s+uses = "docker:\/\/([^"]+)"\s*$/.exec(line) ||
-      /^\s+uses: docker:\/\/([^"]+)\s*$/.exec(line);
-    if (match) {
-      const [, currentFrom] = match;
+    if (line.trim().startsWith('#')) {
+      continue; // eslint-disable-line no-continue
+    }
+
+    const dockerMatch = /^\s+uses: docker:\/\/([^"]+)\s*$/.exec(line);
+    if (dockerMatch) {
+      const [, currentFrom] = dockerMatch;
       const dep = getDep(currentFrom);
       logger.debug(
         {
           depName: dep.depName,
           currentValue: dep.currentValue,
           currentDigest: dep.currentDigest,
         },
-        'Docker image inside GitHub Actions'
+        'Docker image inside GitHub Workflow'
       );
       dep.versioning = dockerVersioning.id;
       deps.push(dep);
+      continue; // eslint-disable-line no-continue
+    }
+
+    const tagMatch = /^^\s+-?\s+?uses: (?<lookupName>[a-zA-Z-_]+\/[a-zA-Z-_]+)(?<path>.*)?@(?<currentValue>.+?)\s*?$/.exec(
```

Comment:
```suggestion
    const tagMatch = /^^\s+-?\s+?uses: (?<depName>[a-zA-Z-_]+\/[a-zA-Z-_]+)(?<path>.*)?@(?<currentValue>.+?)\s*?$/.exec(
```
`depName` should always be present, renovate will fallback to `depName` for lookup, when no `lookupName` is set. So `lookupName` can be obmitted, if it has same value as `depName`.

Reply:
I think numbers are allowed too for github org and repo names?

Reply:
You're correct! Swapping for `\w-`